src/web/blocks.rs

```diff
@@ -52,41 +56,39 @@ pub struct BlockDTO {
     height: u32,
     timestamp: Timestamp,
     block_hash: SerScaleHex<Hash>,
-    parent_block_hash: SerScaleHex<Hash>,
-    transactions_merkle_root_hash: SerScaleHex<Hash>,
-    rejected_transactions_merkle_root_hash: SerScaleHex<Hash>,
+    parent_block_hash: SerScaleHex<Option<HashOf<VersionedCommittedBlock>>>,
+    transactions_merkle_root_hash:
+        SerScaleHex<Option<HashOf<MerkleTree<VersionedSignedTransaction>>>>,
+    rejected_transactions_merkle_root_hash:
+        SerScaleHex<Option<HashOf<MerkleTree<VersionedSignedTransaction>>>>,
     invalidated_blocks_hashes: Vec<SerScaleHex<Hash>>,
-    transactions: Vec<SerScaleHex<VersionedValidTransaction>>,
-    rejected_transactions: Vec<SerScaleHex<VersionedRejectedTransaction>>,
+    transactions: Vec<SerScaleHex<TransactionValue>>,
+    rejected_transactions: Vec<SerScaleHex<VersionedSignedTransaction>>,
     view_change_proofs: Vec<SerScaleHex<Hash>>,
+    // signature: SignaturesOf<CommittedBlock>,
```

Comment:
Why this field is not presented in the DTO anymore?

Also, this change is not reflected in `api.ts`.

Reply:
After analyzing the Figma designs there is no need for a signature type field in blockShallow and blockDetails